go-controller/cmd/ovnkube-trace/README.md

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Sorry I meant this should be put in `ovn-org/ovn-kubernetes/docs` with all the other documentation